Directory.Build.props

```diff
@@ -21,6 +21,7 @@
 
   <ItemGroup Label="Package References">
     <PackageReference Include="Microsoft.CodeAnalysis.FxCopAnalyzers" PrivateAssets="All" Version="2.9.8" />
+    <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies" PrivateAssets="All" Version="1.0.0" />
```

Comment:
Should have done this in the last PR.

Reply:
Ahhhh - yeah, missed that when I went through the files. Good idea!